### PR TITLE
docs: improve section about Prometheus graceful shutdown.

### PR DESCRIPTION
**Pull request by machine424**
fixes https://github.com/prometheus/prometheus/issues/13183

### CODE REVIEW

**Comment by machine424** [comment]
cc @beorn7

File: docs/getting_started.md [hunk 1]
```diff
@@ -262,6 +262,9 @@ process ID.
...
 
 While Prometheus does have recovery mechanisms in the case that there is an
-abrupt process failure it is recommend to use the `SIGTERM` signal to cleanly
-shutdown a Prometheus instance. If you're running on Linux this can be performed
-by using `kill -s SIGTERM <PID>`, replacing `<PID>` with your Prometheus process ID.
+abrupt process failure it is recommended to use signals or interrupts for a
+clean shutdown of a Prometheus instance. On Linux, this can be done by sending
+the `SIGTERM` or `SIGINT` signals to the Prometheus process. For example, you
+can use `kill -s <SIGNAL> <PID>`, replacing `<SIGNAL>` with the signal name
+and `<PID>` with the Prometheus process ID. Alternatively, one can press the
```
Comment (beorn7) [inline docs/getting_started.md:269]:
```suggestion
and `<PID>` with the Prometheus process ID. Alternatively, you can press the
```

Just to keep the "you" voice used before.

**Review by beorn7 — CHANGES_REQUESTED** [review]
Thank you very much, just a small nit.

File: docs/getting_started.md (hunk 1 above)
Comment (machine424) [reply docs/getting_started.md]: Sure.

**Review by beorn7 — APPROVED** [review]

**Pull request merged**